src/adapters/IDB.js

```diff
@@ -427,7 +427,8 @@ function transactionProxy(store, preloaded = []) {
     },
 
     update(record) {
-      store.put(record);
+      // We need to put undefined there so that Safari and IE are happy.
+      store.put(record, undefined);
```

Comment:
Well, I'd be surprised this changes anything as `store.put(record)` and `store.put(record, undefined)` are equivalent in js, where args have `undefined` as a value by default.